packages/arb-bridge-peripherals/package.json

```diff
@@ -32,6 +37,10 @@
     "typechain": "^4.0.0",
     "typescript": "^4.1.5"
   },
+  "dependencies": {
```

Comment:
Is there a particular reason to change these to dependencies rather than devDependency?

Reply:
The hoisting didn't seem to be working w/ them as dev dependencies, but I'll take another look